runtime/jcl/common/clsldr.cpp

```diff
@@ -116,9 +116,12 @@ Java_java_lang_ClassLoader_defineClassImpl1(JNIEnv *env, jobject receiver, jclas
 
 	jclass result = defineClassCommon(env, receiver, className, classRep, 0, length, protectionDomain, &options, hostClazz, NULL, FALSE);
 	if (env->ExceptionCheck()) {
+		throwNewInternalError(env, NULL);
```

Comment:
Why throw internal error if there's a perfectly good exception already pending?

Reply:
I added this when debugging some tests where we missed to set an exception. This should be removed and it is removed now.